components/service/glean/README.md

```diff
@@ -80,19 +80,21 @@ class SampleApplication : Application() {
 ```
 
 Once initialized, if collection is enabled, glean will automatically start collecting [baseline metrics](metrics.yaml)
-and sending its [pings](docs/pings.md).
+and sending its [pings](docs/pings/pings.md).
 
 Glean should be initialized as soon as possible, and importantly, before any
 other libraries in the application start using Glean. Library code should never
 call `Glean.initialize`, since it should be called exactly once per application.
 
-### Defining metrics
+### Adding new metrics
```

Comment:
Can you also update the TOC at the beginning of the file since the name of the section changed?